modules/nw-configuring-clb-timeouts.adoc

```diff
@@ -0,0 +1,42 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-clb-timeouts_{context}"]
+= Configuring Classic Load Balancer timeouts
+
+You can configure the default timeouts for a Classic Load Balancer (CLB) to extend idle connections.
+
+.Prerequisites
+
+* You need a deployed Ingress Controller on a running cluster.
+
+.Procedure
+
+* Using the `oc patch` command, set an AWS connection idle timeout of five minutes for the default `ingresscontroller`:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default \
+    --type=merge --patch='{"spec":{"endpointPublishingStrategy": \
+    {"type":"LoadBalancerService", "loadBalancer": \
+    {"scope":"External", "providerParameters":{"type":"AWS", "aws": \
+    {"type":"Classic", "classicLoadBalancer": \
+    {"connectionIdleTimeout":"5m"}}}}}}}'
+----
++
+To restore the default value of the timeout, run the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontroller/default \
+    --type=merge --patch='{"spec":{"endpointPublishingStrategy": \
+    {"loadBalancer":{"providerParameters":{"aws":{"classicLoadBalancer": \
+    {"connectionIdleTimeout":null}}}}}}}'
+----
+
+[NOTE]
+====
+You must specify the `scope` field when you change the connection timeout value unless the current scope has already been set. When you set the `scope` field, you do not need to do so again when restoring the default timeout value.
```

Comment:
```suggestion
You must specify the `scope` field when you change the connection timeout value unless the current scope is already set. When you set the `scope` field, you do not need to do so again when [after?] you restore the default timeout value.
```

Reply:
Thanks for pointing this out, I see the confusion this sentence creates